nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -1125,6 +1139,14 @@ in
         '';
       }
 
+      {
+        assertion = cfg.package.pname != "nginxQuic" -> !(cfg.enableQuicBPF);
+        message = ''
+          services.nginx.enableQuicBPF requires using nginxQuic package,
+          which can be achieved by setting `services.nginx.package = pkgs.nginxQuic;`.
+        '';
```

Comment:
Should we also assert that at least nginx 1.25 is used? 1.24 does not support this option despite having initial quic support

Reply:
This feature appeared long before version 1.24 in a separate nginx-quic branch.